test/Microsoft.AspNetCore.SignalR.Tests/TestClient.cs

```diff
@@ -28,7 +28,7 @@ public class TestClient : IDisposable, IInvocationBinder
         public Channel<byte[]> Application { get; }
         public Task Connected => ((TaskCompletionSource<bool>)Connection.Metadata["ConnectedTask"]).Task;
 
-        public TestClient(bool synchronousCallbacks = false)
+        public TestClient(bool synchronousCallbacks = false, bool addClaimId = false)
```

Comment:
Having this param permanent will require to fix test case with unauthorized user, since it's currently check that user have no `NameIdentifier` claim (well, testcase not throwing unauthorized exception if user have those claim, but case expect it will be thrown). 

So avoid that kind of parameter will probably require some test code changes and introducing 2 types of clients - authorized and unauthorized. Wasn't sure if that's something to make sense to do, so decide just start with additional parameter.